install_config/aggregate_logging.adoc

```diff
@@ -1101,6 +1101,25 @@ You can see the user interface by visiting the site specified by the
 See the link:https://www.elastic.co/guide/en/kibana/4.5/discover.html[Kibana
 documentation] for more information on Kibana.
 
+[[kibana-visualizations-dashboard]]
+*Kibana Visualizations and Dashboard*
+
+Kibana visualizations and dashboard for monitoring container and pod logs allows
```

Comment:
That "Kibana visualizations and dashboard" wording bug me for some reason. Not sure how to fix it. the Kibana docs have:
"Visualize enables you to create visualizations of the data in your Elasticsearch indices. You can then build dashboards that display related visualizations."
"Kibana visualizations allow you to build dashboards to view logs..." ??